### ISSUE TITLE
Inconsistent validation results

### ISSUE THREAD

**Issue by saltoricristiano** [intel-isl/Open3D-ML#166]
Hi all, thank you very much for the incredible work you have doing.
I'm running inference on the validation split of SemanticKITTI by using the following function I added in the SemanticSegmentation class:

```
def run_valid(self):
        model = self.model
        dataset = self.dataset
        device = self.device
        cfg = self.cfg
        model.device = device
        model.to(device)

        timestamp = datetime.now().strftime('%Y-%m-%d_%H:%M:%S')
        metric = SemSegMetric(self, model, dataset, device)

        log.info("DEVICE : {}".format(device))
        log_file_path = join(cfg.logs_dir, 'log_test_' + timestamp + '.txt')
        log.info("Logging in file : {}".format(log_file_path))
        log.addHandler(logging.FileHandler(log_file_path))

        batcher = self.get_batcher(device, split='validation')

        valid_split = TorchDataloader(dataset=dataset.get_split('validation'),
                                      preprocess=model.preprocess,
                                      transform=model.transform,
                                      use_cache=dataset.cfg.use_cache,
                                      steps_per_epoch=dataset.cfg.get(
                                          'steps_per_epoch_valid', None))

        valid_loader = DataLoader(valid_split,
                                  batch_size=cfg.val_batch_size,
                                  shuffle=True,
                                  collate_fn=batcher.collate_fn)

        self.load_ckpt(model.cfg.ckpt_path)

        datset_split = self.dataset.get_split('validation')

        log.info("Started validation")

        self.valid_accs = []
        self.valid_ious = []

        with torch.no_grad():
            for step, inputs in enumerate(tqdm(valid_loader, desc='validation')):
                
                results = model(inputs['data'])
                labels = inputs['data']['labels']

                pred_scores, gt_labels = filter_valid_label(results, labels, model.cfg.num_classes, 
                                                       model.cfg.ignored_label_inds, device)

                if pred_scores.size()[-1] == 0:
                    continue
                

                acc = metric.acc(pred_scores, gt_labels)
                iou = metric.iou(pred_scores, gt_labels)
                self.valid_accs.append(acc)
                self.valid_ious.append(iou)

            log.info("valid acc: {}".format(
                np.nanmean(np.array(self.valid_accs)[:, -1])))
            log.info("valid iou: {}".format(
                np.nanmean(np.array(self.valid_ious)[:, -1])))
```
Everything works with RandLA but I have inconsistent results with 1) what reported in the table and 2) between consecutive runs.
These are the results I get between 3 consecutive runs:
```
RUN 1
valid acc: 0.6399258845541128
valid iou: 0.4675712403922625
```

```
RUN 2
valid acc: 0.638801616699286
valid iou: 0.46817005750227314
```

```
RUN 3
valid acc: 0.6440866445326203
valid iou: 0.48439236029086935
```

I tried to run by setting the same np.random.seed(seed=1024), but nothing changed, still different results.
Is these anything I'm missing? Any solution? Which split do the numbers for the pre-trained models refer to?
Another thing I noticed is that checksum differ from the one you report on the website. I downloaded several times and always got the same checksum but still different from the reported one.

**Comment by saltoricristiano** [comment]
I found the problem and solved it, therefore I'm going to close the issue!

**Comment by Hao-HUST** [comment]
> I found the problem and solved it, therefore I'm going to close the issue!

Hi, how did you solved this issue?

**Comment by Hao-HUST** [comment]
> I found the problem and solved it, therefore I'm going to close the issue!

I met the same problem, and it is more strange for me. With the same evaluation code for the Semantic-Kitti validation dataset, my valid acc is 0.71595 and my valid iou is 0.3443, which are far away from both your value and reported performance(52.8). 

Do you have a suggestion about that?

**Comment by KlyachevDmitry** [comment]
> I found the problem and solved it, therefore I'm going to close the issue!

Hello, please write how you solved the problem, I also have a different output
